src/pages/SignIn/components/UserSignIn.jsx

```diff
@@ -1,17 +1,18 @@
 import { signInWithEmailAndPassword } from 'firebase/auth';
-import React, { useState } from 'react';
+import React, { useState, useRef } from 'react';
 import { auth } from '../../../utils/firebase';
+import { Input, Button } from '@nextui-org/react';
 
 const UserSignIn = () => {
-  const [email, setEmail] = useState('');
-  const [password, setPassword] = useState('');
+  const email = useRef();
+  const password = useRef();
   const signIn = (e) => {
     e.preventDefault();
-    signInWithEmailAndPassword(auth, email, password)
+    signInWithEmailAndPassword(auth, email.current.value, password.current.value)
       .then((userCredential) => {
         console.log(userCredential);
-        setEmail('');
-        setPassword('');
+        email.current.value = '';
+        password.current.value = '';
```

Comment:
i think you dont need this part. once user is defined, the page will show profile page

Reply:
ur right. done